app/assets/javascripts/media_player_wrapper/mejs4_link_back.es6

```diff
@@ -0,0 +1,161 @@
+          media.addEventListener('play', linkBackObj.handlePlay.bind(t));
+          // Set up listener for showing title link on pause
+          media.addEventListener('pause', linkBackObj.handlePause.bind(t));
+        }
+    },
+
+    // Optionally, each feature can be destroyed setting a `clean` method
+
+    /**
+     * Feature destructor.
+     *
+     * Always has to be prefixed with `clean` and the name that was used in MepDefaults.features list
+     * @param {MediaElementPlayer} player
+     * @param {HTMLElement} controls
+     * @param {HTMLElement} layers
+     * @param {HTMLElement} media
+     */
+    cleanlinkBack (player, controls, layers, media) {
+    },
+
+    // Other optional public methods (all documented according to JSDoc specifications)
+
+    /**
+     * The 'linkBackObj' object acts as a namespacer for this plugin's
+     * specific variables and methods, so as not to pollute global or MEJS scope.
+     * @type {Object}
+     */
+    linkBackObj: {
+      titleLinkEl: null,
+      controlButtonEl: null,
+
+      createTitleLinkEl: function () {
+        let linkBackObj = this.linkBackObj;
+        let titleLinkEl = document.getElementById('mejs-title-link');
+
+        // If element already exists, remove it
+        if (titleLinkEl) {
+          titleLinkEl.parentNode.removeChild(titleLinkEl);
+        }
+        titleLinkEl = document.createElement('div');
+        titleLinkEl.classList.add('mejs__title-link');
+        titleLinkEl.id = 'mejs-title-link';
+        titleLinkEl.innerHTML = '<a href="' + linkBackObj.player.options.link_back_url + '">' + linkBackObj.player.options.embed_title + '</a>';
+        linkBackObj.controls.parentNode.insertBefore(titleLinkEl, linkBackObj.controls);
+
+        return titleLinkEl;
+      },
+
+      createControlButtonEl: function () {
+        const t = this;
+        let linkBackObj = this.linkBackObj;
+        let controlButtonEl = document.getElementById('mejs-link-back');
+
+        // If element already exists, remove it
+        if (controlButtonEl) {
+          controlButtonEl.parentNode.removeChild(controlButtonEl);
+        }
+
+        const addTitle = 'View in Repository';
+        controlButtonEl = document.createElement('div');
+        controlButtonEl.className = t.options.classPrefix + 'button ' + t.options.classPrefix + 'link-back-button';
+        controlButtonEl.innerHTML = `<button type="button" aria-controls="${t.id}" title="${addTitle}" aria-label="${addTitle}" tabindex="0">${addTitle}</button>`;
+
+        return controlButtonEl;
+      },
+
+      /**
+       * Handle control button click to open Avalon in a separate browser window/tab
+       * @param  {MouseEvent} e Event generated when control button clicked
+       * @return {void}
+       */
+      handleControlClick: function (e) {
+        let linkBackObj = this.linkBackObj;
+        return window.open(linkBackObj.player.options.link_back_url);
```

Comment:
In general (micro-optimization...probably not a big deal), only create a locally scoped variable if referencing more than once in a function.